internal/staticsources/unix/source.go

```diff
@@ -50,7 +60,7 @@ func (s *Source) Run(params defs.StaticSourceRunParams) error {
 	}
 	defer socket.Close()
 
-	conn, err := socket.Accept()
+	conn, err := acceptWithContext(socket, time.Duration(10)*time.Second)
```

Comment:
```suggestion
	conn, err := acceptWithContext(socket, 10*time.Second)
```

Since 10 is a compile time constant you don't need to explicitly cast it.